tool/grind.dart

```diff
@@ -50,7 +50,7 @@ Future<void> serveLocalAppEngine() async {
   ConstTaskArgs('build', flags: {
     _debugFlag: true,
   }, options: {
-    _serverUrlOption: 'http://127.0.0.1:8084/',
```

Comment:
@johnpryan was this set to 8084 for a reason? I feel like this change should be safe, but I don't know DartPad well enough to know for sure.